signin/index.js

```diff
@@ -0,0 +1,106 @@
+// email input 포커스 아웃을 했는데 값이 없을때
+let email = document.querySelector('#email');
+let form = document.querySelector('.contents-form');
```

Comment:
DOM 에서 가져오는 것들이라 따로 변경할 게 없어서 이 부분은 const 로 선언해주시는 게 더 좋을 거 같아요!
var 보다는 무조건 let 이 낫고, 바뀌지 않는 값의 경우 let 보다는 const 가 낫습니다!
```suggestion
const email = document.querySelector('#email');
const form = document.querySelector('.contents-form');
```